### ISSUE TITLE
Qualified names in tuple in except not handled by error analysis

### ISSUE THREAD

**Issue by staticdev** [terrencepreilly/darglint#119]
Hi again,

I have just added darglint to a new repo and I have no `.darglint` file, and `.flake8` don't have DAR selected. But when it runs, I get many DAR errors.

I am also getting there strange errors:

```
ERROR:darglint:While getting the names from a caught tuple of exceptions, encountered something other than an ast.Name: Attribute
ERROR:darglint:While getting the names from a caught tuple of exceptions, encountered something other than an ast.Name: Attribute
```

How to reproduce:

with the code in https://github.com/staticdev/git-portfolio/tree/a0031294ff5a5a20694f6c13d46e94f5f10cb687 run:

```
poetry shell
poetry install
pre-commit run --all-files
```

**Comment by terrencepreilly** [comment]
The many actual errors appear to be legitimate. If you'd like to silence them, you can add the following to your `.flake8` file:

```
[darglint]
strictness=short
````

Which will allow docstrings like 

```
   def _create_issue_from_repo(self, github_repo, issue, labels):
        """Create issue from one repository."""
        ...
```

Which are being reported because there are undocumented arguments.

The other errors, though, are actual errors in darglint's error analysis.  It looks like it's coming from this:

```
    def get_github_username(
        self,
        user: Union[
            github.AuthenticatedUser.AuthenticatedUser, github.NamedUser.NamedUser
        ],
    ) -> str:
        """Get Github username."""
        try:
            return user.login
        except (github.BadCredentialsException, github.GithubException):
            return ""
        except requests.exceptions.ConnectionError:
            sys.exit("Unable to reach server. Please check you network.")
```

So, it must not be handling qualified names within tuples in an except statement.  Thanks for finding that!  It should be an easy fix.

**Comment by terrencepreilly** [comment]
Should be resolved in v1.5.3

**Comment by staticdev** [comment]
@terrencepreilly working perfectly, thanks!
